book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(person, date, self)
+  end
```

Comment:
- [ ] Kindly rearrange your method here to match the order of parameters in the rental constructor.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```